net-fs/s3fs/s3fs-1.85.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+MY_PN=${PN}-fuse
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="Amazon S3 mounting via fuse"
+HOMEPAGE="https://github.com/s3fs-fuse/s3fs-fuse/"
+SRC_URI="https://github.com/${MY_PN}/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+openssl nss gnutls nettle"
+REQUIRED_USE="
+	^^ ( openssl nss gnutls )
+	nettle? ( gnutls )"
+
+CDEPEND="
```

Comment:
`CDEPEND` is semi-banned, we try not to use it. `COMMON_DEPEND` is much more preferred nowadays.

Reply:
I redone dependencies in such a way that we don't need `COMMON_DEPEND` anymore. But thanks, I will remember that for next time.